### DISCUSSION TITLE
Have you seen this project of VSCode integration for Supabase?

### DISCUSSION THREAD

**Discussion by Zebnastien** [supabase/supabase#27151] [Show and tell]
I was searching for a way to integrate Supabase Database Functions with VSCode when I stumbled upon this fantastic project by @anas-araid.

Check out the project demo video here:
https://github.com/anas-araid/vscode-supabase-cli

Note: I have no affiliation with this project nor the author; I just thought it was worth sharing.

**Comment by Jalal-Nasser** [comment]
I was looking for something like this, thank you
